packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -113,6 +113,41 @@ Form submit buttons should be disabled until a user starts filling out the form
 * For a submitted form, disable the submit button while a user's responses are processing to avoid a duplicate submission.
 * For a processing form, use feedback messages and progress indicators like spinners or progress bars if the processing time might exceed user expectations.
 
+## Form sections
+Form sections sort form fields into non-hierarchical groups based on common information type, step, or visual preference. PatternFly form sections support a header followed by a left-aligned list of unnested and uncollapsible form groups, so they create a relatively flat information hierarchy. Form sections are ideal for breaking lengthy forms into simple, evenly spaced segments with all information exposed. 
+
+Default to grouping data with form sections, especially if all form fields are required. If you need to visualize complex or nested information, use [field groups](#field-groups) instead.
+
+To distinguish between form sections, title them based on the reason you grouped their inputs together. You may forego form section titles, but make sure visual separation isn't incredibly meaningful to your content. To keep form sections accessible, add an `aria-label` to indicate why you're sectioning each set of form fields together.
+
+Example: For a form section that allows users to configure traffic between pods by specifying origins, ports, and specific rules, add a title like *Ingress rules*.
+
+<img src="./img/form-sections.png" alt="Example of an 'Ingress rules' form section that groups radio buttons, dropdowns, and actions together. Form sections can feature interactive behavior typical to normal form fields, including progressive disclosure, surfacing fields based on user input." width="742"/>
```

Comment:
I feel like this image is confusing and looks like it refers to field groups more than it does form fields.
Could we use a simpler image, with just the headers, and then very simple/normal form fields underneath?